Octokit.Tests.Integration/Clients/RepositoriesClientTests.cs

```diff
@@ -1697,57 +1697,42 @@ public class TheTransferByNameMethod
                 var transferred = await github.Repository.Get(newOwner, context.RepositoryName);
             }
         }
-
+        
         [IntegrationTest]
-        public async Task CanTransferUserRepoToOrg()
+        public async Task TransfersFromOrgToUserWithId()
```

Comment:
to be consistent with your other method names, this should be `ById` rather than `WithId`

Reply:
Oops.